src/Esprima/Utils/ToJavascriptConverter.cs

```diff
@@ -0,0 +1,1410 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Esprima.Ast;
+
+namespace Esprima.Utils
+{
+    public class ToJavascriptConverter
+    {
+        public static string ToJavascript(Node node)

```

Comment:
This should be on the parsed tree directly

Reply:
don't know what you mean...
i've added a ToJavascriptConverterExtension class with an extension method. is this what you meant?